src/common/components/confirmMutationButton/ConfirmMutationButton.tsx

```diff
@@ -58,7 +58,7 @@ const ConfirmMutationButton = ({
         key="button"
         className={className}
       >
-        {Icon && Icon}
+        {Icon}
```

Comment:
Icon can be undefined, so would this render `undefined` in that case?

Reply:
I think not because https://react.dev/learn/conditional-rendering
> React considers false as a “hole” in the JSX tree, just like null or undefined, and doesn’t render anything in its place.